case-lib/hijack.sh

```diff
@@ -7,6 +7,11 @@ function exit()
 {
     local exit_status=${1:-0}
 
+    # To generate topology graph, test case should export $tplg for us
+    tplgtool.py -d graph -D $LOG_ROOT $tplg > /dev/null
+    [[ "$?" != "0" ]] && dloge "Failed to generate topology graph"
+    unset tplg
+
```

Comment:
this place is good but who pass-down the `$tplg`  
almost our case define the `tplg` but still have case not define it or not use this name  
how about move it into `func_pipeline_export` here you can got the TPLG file name  
almost of our case need TPLG will load this case

Reply:
I want to add a feature to highlight the pipeline we are testing on when error occurs later, so I think putting the generation of tplg graph at the exit function is a better way. can we modify all the cases to define tplg?
On who pass down $tplg: If case want to exit, the case export $tplg and some other infomation, eg, test error pipeline, to us?

Reply:
> I want to add a feature to highlight the pipeline we are testing on when error occurs later, so I think putting the generation of tplg graph at the exit function is a better way. can we modify all the cases to define tplg?  

yes, so you need to check whether each use TPLG case already define tplg   
>On who pass down $tplg: If case want to exit, the case export $tplg and some other infomation, eg, test error pipeline, to us?

you also need to add `tplg` is not define logic  
because not define is allow for some case like 'check-sof-logger.sh' etc.